test/escaping-support.spec.ts

```diff
@@ -0,0 +1,10 @@
+import {HyperFormula} from '../src'
+import {adr} from './testUtils'
+
+describe('escaped formulas', () => {
+  it('should serialize properly', () => {
+    const engine = HyperFormula.buildFromArray([['\'=SUM(2,2)']], {licenseKey: 'non-commercial-and-evaluation'})
```

Comment:
I think that `licenseKey` is a part of the default config for tests? Not needed here